app/src/main/java/com/liato/bankdroid/TransactionsActivity.java

```diff
@@ -66,7 +66,7 @@ public void onCreate(Bundle savedInstanceState) {
         viewBankName.setText(bank.getDisplayName());
         viewAccountName.setText(account.getName());
         viewAccountBalance
-                .setText(Helpers.formatBalance(account.getBalance(), account.getCurrency()));
+                .setText(Helpers.formatBalance(account.getBalance(), account.getCurrency(), false));
```

Comment:
Not a strong opinion, but since this particular variant is always called with `false` it makes sense to skip the parameter here and go for false. And it was just one refactoring away. Done!